apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.js

```diff
@@ -178,6 +185,25 @@ router.post('/', function (req, res, next) {
         }
       });
     });
+  } else if (req.body.__action === 'unsafe_sharing_set_add') {
+    debug('Add question to sharing set');
+    const questionSharingEnabled = features.enabledFromLocals('question-sharing', res.locals);
+    if (!questionSharingEnabled) {
+      throw error.make(403, 'Access denied (feature not available)');
```

Comment:
You need to either a) use `next(...)` instead of `throw` or b) refactor this handler to use `express-async-handler`.

Reply:
done, thanks